manifests/init.pp

```diff
@@ -8,8 +8,8 @@
 # @example
 #   include tang
 class tang (
-  Array[String[1]] $package_name,
-  String[1]        $service_name,
+  Array[String[1]] $package_name = ['tang'],
```

Comment:
Do I want to know why this is an `Array[String]`?

Reply:
to allow multiple packages in the future I assume. We've the same design in the clevis module.